package.json

```diff
@@ -1,5 +1,17 @@
 {
   "name": "@patternfly/patternfly-elements",
+  "publishConfig": {
```

Comment:
The publishConfig can be removed, it's only needed in the element's package files.  It allows the package to be published to npm, and we don't want to publish the entire monorepo to npm.

Reply:
+1 fixing that now